lib/csu/Makefile.inc

```diff
@@ -8,15 +8,62 @@ NO_WMISSING_VARIABLE_DECLARATIONS=
 
 .if !defined(BUILDING_TESTS)
 
+OBJS+=	Scrt1.o crt1.o
 OBJS+=	crtbegin.o crtbeginS.o crtbeginT.o
 OBJS+=	crtend.o crtendS.o
+.if !${MACHINE_ABI:Mpurecap}
+OBJS+=	gcrt1.o
+OBJS+=	crti.o crtn.o
+.endif
+
+CRT1OBJS+=	crtbrand.o ignore_init_note.o
 
 ACFLAGS+=	-DLOCORE
 
 CFLAGS+=	-fno-asynchronous-unwind-tables
 CFLAGS+=	-fno-omit-frame-pointer
+CFLAGS+=	-I${.CURDIR:H}/common \
+		-I${SRCTOP}/lib/libc/include
+
+.if ${MACHINE_ABI:Mpurecap}
+# Purecap code requires crt_init_globals to be called before using jump tables
+CFLAGS+=	-fno-jump-tables
+CFLAGS+=	-DCRT_IRELOC_SUPPRESS
```

Comment:
I had left this one in each file, but it's fine to do here.

Reply:
With just riscv and mips I'd have kept it there, but with morello on the way and also using these flags it felt like a little clutter here was worth it.